src/main/java/com/kjetland/dropwizard/activemq/SessionCallback.java

```diff
@@ -0,0 +1,10 @@
+package com.kjetland.dropwizard.activemq;
+
+import javax.jms.JMSException;
+import javax.jms.Session;
+
+@FunctionalInterface
+public interface SessionCallback<T> {
+
+    T withSession(Session session) throws JMSException;
```

Comment:
Same comment as in the other callback, feelt like a verb is needed in the name of the method